app/assets/stylesheets/nav.scss

```diff
@@ -0,0 +1,9 @@
+.search-bar span {
+  display: inline-block;
+}
+
+.search-button {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.